boards/shields/x_nucleo_wb05kn1/Kconfig.defconfig

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_X_NUCLEO_WB05KN1_UART || SHIELD_X_NUCLEO_WB05KN1_SPI
+
+if BT
+
+config SPI
+	default y
+	depends on SHIELD_X_NUCLEO_WB05KN1_SPI
+
+config SPI_STM32_INTERRUPT
+	default y
+	depends on SPI
+
+choice BT_HCI_BUS_TYPE
+	default BT_SPI if DT_HAS_ST_HCI_SPI_V2_ENABLED
+	default BT_H4
+endchoice
```

Comment:
I think this needs to be updated due to changes in https://github.com/zephyrproject-rtos/zephyr/pull/72323 as BT_HCI_BUS_TYPE has been removed.

Reply:
Done.